libs/blocks/caas-config/caas-config.css

```diff
@@ -36,3 +36,72 @@ dd.content .sort-options > .field {
   top: 12px;
 }
 
+.config-panel dd.content.is-expanded {
+  max-height: 6000px;
+}
+
+.multifield .multifield {
+  padding: 0 0 15px;
+  margin: 0 -15px 9px 0;
```

Comment:
Negative margins are usually bad practice, if you can avoid them anyhow.

Reply:
We try to avoid negative margins when possible,